IDEAS/Fluid/Taps/BaseClasses/PartialTap.mo

```diff
@@ -0,0 +1,240 @@
+          thickness=0.5)}),
+    Documentation(info="<html>
+<p><b>Description</b></p>
+<p>Partial model of a domestic hot water (DHW) tap. The tap is modelled as a thermostatic mixing valve.</p>
+<p>The model has two flowPorts and a realInput: </p>
+<ul>
+<li><i>port_hot</i>: connection to the hot water source (designation: <i>hot</i> )</li>
+<li><i>port_cold</i>: connection to the inlet of cold water in the hot water source (designation: <i>cold </i>)</li>
+<li><i>m_flow_Set</i>: desired flowrate of DHW water, equivalent at a user defined set point temperature</li>
+</ul>
+<p>The model tries to reach the given DHW flow rate at a the desired mixing temperature <i>TSet </i>by mixing the hot water with cold water. The resulting hot flowrate (<i>m_flow_Hot </i>) will be extracted automatically from the hot source (via <i>port_hot </i>). This same flow rate will be injected at <i>TCold</i> in the production system through the connection of <i>port_cold</i> to the hot source. </p>
+<p><b>Assumptions and limitations </b></p>
+<ol>
+<li>No heat losses.</li>
+<li>Inertia is foreseen through the inclusion of a water volume on the hot water side (default=1 m3). <br>This parameter is not propagated to the interface, but it can be changed by modifying vol.V. <br>Putting this water content to zero may lead to numerical problems (not tested)</li>
+<li>If <i>THot</i> is smaller than <i>TSet</i>, there is no mixing and <i>TMixed</i> = <i>THot</i>.</li>
+<li>Fixed <i>TSet</i> and <i>TCold </i>as parameters.</li>
+<li>The mixed DHW is not available as an outlet or flowPort. It is assumed to be &apos;consumed&apos;. </li>
+</ol>
+<p><b>Model use</b></p>
+<ol>
+<li>Set the parameters for cold water supply temperature <i>TCold</i> and the DHW set temperature <i>TSet</i> (mixed).</li>
+<li>Connect <i>port_hot </i>to the hot water source.</li>
+<li>Connect <i>port_</i>c<i>old</i> to the cold water inlet of the hot water source.</li>
+<li>Depending on the implementation: fill out the table or provide a realInput for <i>m_flow_Set.</i></li>
+<li>Thanks to the use of an <a href=\"IDEAS.Fluid.Interfaces.IdealSource\">IdealSource</a> in this model, it is <b>NOT</b> required to add additional pumps, ambients or AbsolutePressure to the DHW circuit.</li>
+</ol>
+<p><b>Validation </b></p>
+<p>The model is verified to work properly by simulation of the different operating conditions.</p>
+<p><b>Examples</b></p>
+<p>An example of this model is given in <a href=\"IDEAS.Fluid.Taps.Examples.DHW_example\">IDEAS.Fluid.Taps.Examples.DHW_example</a>.</p>
+</html>", revisions="<html>
+<ul>
+<li>March 26, 2024, by Lucas Verleyen:<br>Adaptations to make the model more clear (change variable names, set allowFlowReversal=false, update icon).<br>Include a user-defined DHW setpoint temperature.<br>Add indicator for comfort or discomfort at DHW tap.<br>Update documentation. <br>See <a href=\"https://github.com/open-ideas/IDEAS/issues/1287\">#1287</a> for more information. </li>
```

Comment:
I would keep this shorter, everything you changed can be consulted in the IDEAS issue to which you refer.

Reply:
ok!